presto-main/src/main/java/com/facebook/presto/metadata/InternalNodeManager.java

```diff
@@ -32,4 +33,8 @@
     AllNodes getAllNodes();
```

Comment:
Aside: The name `AllNodes` really bugs me for some reason. Can't remember any similarly named class in Presto.